jenkins/hourly_maintenance.py

```diff
@@ -194,6 +196,29 @@ def DeleteCorruptGitRepos():
     return err
 
 
+def DeleteOldWorkspaces():
+    err = 0
+    live_jobs = None
+    for host in ('jenkins-master', 'pull-jenkins-master'):
```

Comment:
Hmm... I suspect one of these will always fail? Specifically I don't think the light agents have access to pull-jenkins-master and the pr agents jenkins-master

Oh I guess you're just ignoring these errors :neutral_face: 

Somewhat gross but you could theoretically check whether 'agent-pr' is in the host name... :nauseated_face: